tools/build_browser.js

```diff
@@ -166,6 +166,9 @@ async function buildBrowserHighlightJS(languages, { minify }) {
 
   const tasks = [];
   tasks.push(fs.writeFile(outFile, fullSrc, { encoding: "utf8" }));
+  tasks.push(fs.writeFile(`${process.env.BUILD_DIR}/package.json`, JSON.stringify({
+    type:'commonjs'
+  }), { encoding: "utf8" }));
```

Comment:
In what cases would this be used?  

We already have a `cdn` build target that publishes a `package.json`... I wonder if it's not this package.json that perhaps needs updating?  Are browsers really looking for `package.json` or is this more informational?

Typically one builds `browser` just to get an individual JS file that they plan to copy elsewhere - it's not part of a "package" in any real sense - it's never published to `npm`, etc.

Reply:
I added it because the tests with `browser -n` were failing (it was trying to load the UMD as ESM, which cannot work), that's another perk of having `"type":"module`` on the root `package.json`. On the other hand, it's a very small file and were don't have to include it in the distributable archive, it just needs to be there to run the test suite.

Reply:
Understood.  I just need to double check that this isn't interfering with the CDN build's package.json.